stdlib/public/TensorFlow/Tensor.swift

```diff
@@ -536,10 +536,12 @@ public extension Tensor where Scalar : Numeric {
   @_inlineable @inline(__always)
   init(oneHotAtIndices indices: Tensor<Int32>, depth: Int32,
        onValue: Scalar = 1, offValue: Scalar = 0, axis: Int = -1) {
-    self.init(
-      handle: #tfop("OneHot", indices, Tensor<Int32>(depth),
-                    Tensor(onValue), Tensor(offValue),
-                    axis: axis, T: Scalar.self, TI: Int32.self)
+    self = Raw.oneHot(
+      indices: indices,
+      depth: Tensor<Int32>(depth),
+      onValue: Tensor(onValue),
+      offValue: Tensor(offValue),
+      axis: Int64(axis)
```

Comment:
That's a good point and applicable to actually all swift attributes that we generate in the bindings, Int64 is used there as the [attr_value proto](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/framework/attr_value.proto) uses int64.

Reply:
My impression is that int64 indices won't work well with TPU, which needs to be tested once we downstream this change back to google internal. Moreover, int64 can be very inconvenient because the size type is int32 by default, in `TensorShape`. Int32 at least guarantees universal compatibility across all device, until future generations of hardware favor int64, in which case we can just use `Int` uniformly in a Swift for TF program.